docs/podman-rm.1.md

```diff
@@ -24,6 +24,13 @@ Remove all containers.  Can be used in conjunction with -f as well.
 Instead of providing the container name or ID, use the last created container. If you use methods other than Podman
 to run containers such as CRI-O, the last started container could be from either of those methods.
 
+**--sync**
+
+Force a sync of container state with the OCI runtime before attempting to remove.
```

Comment:
Might just be me, but when reading this I'm left wondering what "remove" pertains to.  Container states, OCI runtime, or the container?  Perhaps "remove" to "remove the container"?